bokehjs/examples/stocks/stocks.ts

```diff
@@ -0,0 +1,74 @@
+import plt = Bokeh.Plotting;
+
+console.log(`Bokeh ${Bokeh.version}`);
+Bokeh.set_log_level("debug");
+
+
+function make_plot(title:string, source:Bokeh.ColumnDataSource) {
+    
+    // Create plot
+    const xr = new Bokeh.DataRange1d({});
+    const yr = new Bokeh.DataRange1d({});
+    const plot = new plt.Figure({
+        tools: "pan,wheel_zoom,save,reset",        
+        x_range: xr,
+        y_range: yr,         
+        title: title,
+        plot_width: 400,
+        plot_height: 400,
+        background_fill_color: "#eeeeff",
+    });
+    
+    // Add axis and grid
+    const xaxis = new Bokeh.DatetimeAxis({axis_line_color: null, axis_label: 'time'});
+    const yaxis = new Bokeh.LinearAxis({axis_line_color: null, axis_label: 'price'});
+    plot.add_layout(xaxis, "below");
+    plot.add_layout(yaxis, "left");
+    plot.add_layout(new Bokeh.Grid({ticker: xaxis.ticker, dimension: 0}));
+    plot.add_layout(new Bokeh.Grid({ticker: yaxis.ticker, dimension: 1}));
+    
+    // Add a line for each entry in the source
+    const colors = ['#aa0000','#00aa00', '#0000aa', '#aaaa00', '#aa00aa', '#00aaaa'];
+    let i = -1;
+    for (let key in source.data) {
+        if (key != 't') {
+            i += 1;
+            let line = new Bokeh.Line({x:{field:'t'}, y:{field:key}, legend: key, 
+                                       line_color: colors[i%6], line_width: 2});
```

Comment:
Sorry, the example uses a Figure now. I meant that I do not seem to be able to pass line_color et al. to the `line()` method.